lib/core/text/abstract_text.nit

```diff
@@ -1245,6 +1246,7 @@ abstract class String
 
 	redef fun to_s do return self
 
+	redef fun clone do return self
```

Comment:
That definition should be in `String` since these are `String`s that are guaranteed to be immutable.